client/app/pages/queries/QueryView.jsx

```diff
@@ -159,18 +157,18 @@ function QueryView(props) {
           </div>
         )}
         <div className="query-results m-t-15">
-          {queryResult && queryResultData.status !== "done" && (
-            <div className="query-alerts m-t-15 m-b-15">
+          {(executionError || isExecuting) && (
+            <div style={{ position: "absolute", bottom: "75px", right: "20px", zIndex: 1000 }}>
               <QueryExecutionStatus
-                status={queryResultData.status}
-                updatedAt={queryResultData.updatedAt}
-                error={queryResultData.error}
+                status={executionStatus}
+                error={executionError}
                 isCancelling={isExecutionCancelling}
                 onCancel={cancelExecution}
+                updatedAt={updatedAt}
               />
             </div>
           )}
-          {(!queryResult || queryResultData.status === "done") && (
+          {true && (
```

Comment:
If we indeed keep viz always visible, need to remove this line.

Reply:
I think this is probably the best option, the query view page just look so empty when loading and it makes the whole process of updating results look smoothier :)